doc/whats_new/v1.1.rst

```diff
@@ -388,6 +446,7 @@ Changelog
   :pr:`21481` by :user:`Guillaume Lemaitre <glemaitre>`.
 
 - |Enhancement| :func:`orthogonal_mp_gram` preservse dtype for `numpy.float32`.
+  :pr:`22002` by :user:`Takeshi Oura <takoika>`
```

Comment:
There are a few of these merge issues in this PR:

```suggestion
```

Reply:
Sorry for the merge mess here, I'll fix it this week.